packages/wrangler/templates/middleware/middleware-mock-analytics-engine.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference path="middleware-mock-analytics-engine.d.ts"/>
+
+import { bindings } from "config:middleware/mock-analytics-engine";
+import type { Middleware } from "./common";
+
+const bindingsEnv = bindings.reduce(
+	(acc, binding) => {
+		acc[binding] = {
+			writeDataPoint() {
+				// no op in dev
+			},
+		};
+		return acc;
+	},
+	{} as Record<string, AnalyticsEngineDataset>
+);
```

Comment:
maybe something like?
```suggestion
const bindingsEnv = Object.fromEntries(bindings.map(
	binding => [binding, {
			writeDataPoint() {
				// no op in dev
	}])
);
```